pkg/domain/infra/tunnel/containers.go

```diff
@@ -69,13 +69,17 @@ func (ic *ContainerEngine) ContainerPause(ctx context.Context, namesOrIds []stri
 }
 
 func (ic *ContainerEngine) ContainerUnpause(ctx context.Context, namesOrIds []string, options entities.PauseUnPauseOptions) ([]*entities.PauseUnpauseReport, error) {
+	reports := []*entities.PauseUnpauseReport{}
```

Comment:
Why move this?

Reply:
Maybe because the `len(ctrs)` is not needed anymore so it can move up for readability?

Reply:
I moved it to match containersstop, but if it offends I can move it back